rstcheck.check has been removed

**Describe the bug**
```python

    def test_json_2_rst():
        result = _create_docs(json_schema)
>       assert not list(rstcheck.check(result))
E       AttributeError: module 'rstcheck' has no attribute 'check'
```

**To Reproduce**
Run doc tests

**Expected behavior**
Should pass

--- a/test_requirements.txt
+++ b/test_requirements.txt
@@ -6,5 +6,5 @@ pytest
 pytest-console-scripts
 pytest-httpserver
 pytest-snapshot
-rstcheck<6.0.0
+rstcheck-core
 xlwt

--- a/tests/docs_utils/test_json_schema_2_rst.py
+++ b/tests/docs_utils/test_json_schema_2_rst.py
@@ -1,10 +1,10 @@
-import rstcheck
 import pytest
+import rstcheck_core.checker
 
 from semeio._docs_utils._json_schema_2_rst import (
     _create_docs,
-    _replace_key,
     _remove_key,
+    _replace_key,
 )
 
 json_schema = {
@@ -149,7 +149,7 @@ json_schema = {
 
 def test_json_2_rst():
     result = _create_docs(json_schema)
-    assert not list(rstcheck.check(result))
+    assert not list(rstcheck_core.checker.check_source(result))
 
 
 @pytest.mark.parametrize(

--- a/tests/jobs/correlated_observations_scaling/unit/test_examples.py
+++ b/tests/jobs/correlated_observations_scaling/unit/test_examples.py
@@ -1,6 +1,6 @@
 import configsuite
 import pytest
-import rstcheck
+import rstcheck_core.checker
 
 from semeio.workflows.correlated_observations_scaling import cos, job_config
 
@@ -38,4 +38,4 @@ def test_valid_rst(input_rst):
     Check that the documentation passed through the plugin system is
     valid rst
     """
-    assert not list(rstcheck.check(input_rst))
+    assert not list(rstcheck_core.checker.check_source(input_rst))

--- a/tests/jobs/csv_export2/test_integration.py
+++ b/tests/jobs/csv_export2/test_integration.py
@@ -2,12 +2,11 @@
 import os
 import subprocess
 
-import pytest
-import rstcheck
 import pandas as pd
+import pytest
+import rstcheck_core.checker
 
 from semeio.workflows.csv_export2 import csv_export2
-
 from tests.jobs.csv_export2 import conftest
 
 test_header = [
@@ -129,7 +128,7 @@ def test_valid_rst(input_rst):
     Check that the documentation passed through the plugin system is
     valid rst
     """
-    assert not list(rstcheck.check(input_rst))
+    assert not list(rstcheck_core.checker.check_source(input_rst))
 
 
 @pytest.mark.usefixtures("norne_mocked_ensembleset")

--- a/tests/jobs/localisation/test_valid_rst.py
+++ b/tests/jobs/localisation/test_valid_rst.py
@@ -1,5 +1,5 @@
 import pytest
-import rstcheck
+import rstcheck_core.checker
 
 from semeio.workflows.localisation.local_config_script import DESCRIPTION, EXAMPLES
 
@@ -10,4 +10,4 @@ def test_valid_rst(rst_text):
     Check that the documentation passed through the plugin system is
     valid rst
     """
-    assert not list(rstcheck.check(rst_text))
+    assert not list(rstcheck_core.checker.check_source(rst_text))

